### PR TITLE
1.0.6

### PR DESCRIPTION
**Pull request by Bildcraft1**
👍

### CODE REVIEW

File: RiiConnect24Patcher.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (Bildcraft1) [inline RiiConnect24Patcher.sh]: this run only on elementary and other distros

Comment (larsenv) [reply RiiConnect24Patcher.sh]: Hmm... I don't know much about how the first line works, I think it specifies the shell to use, but are you sure this won't break other OS? I guess it'd be fine...

**Comment by Bildcraft1** [comment]
i think yes

**Comment by Bildcraft1** [comment]
u can upload it?

**Comment by KcrPL** [comment]
We'll test it and if it will pass tests, it will be released as an update ;)

**Comment by larsenv** [comment]
Just reviewed it and it looks fine. Why would you close it?

**Pull request merged**